src/plugins/dashboard/public/dashboard_actions/filters_notification_popover.tsx

```diff
@@ -40,6 +40,7 @@ export function FiltersNotificationPopover({
 }: FiltersNotificationProps) {
   const { embeddable } = context;
   const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  const [disableEditbutton, setDisableEditButton] = useState(false);
```

Comment:
ℹ️ as we are going to give the option to change the ESQL query on the fly we don't want to navigate users to Lens for the text based languages